pkg/fsutil/fsutil_unix.go

```diff
@@ -0,0 +1,11 @@
+//go:build !windows && !plan9
+// +build !windows,!plan9
+
+package fsutil
+
+import "os"
+
+// IsExecutable returns true if the stat object refers to an executable file on UNIX platforms.
```

Comment:
When exporting a function with platform-dependent implementations, I'd prefer putting the entry point in a shared file, even if it is just a call to the unexported version. This gives you a unified Godoc so you don't have to repeat things and don't have to skip between the platforms (see the platform dropdown on e.g. https://pkg.go.dev/golang.org/x/sys/unix#section-documentation) when reading the Godoc to get the whole picture.